kratos/includes/node.h

```diff
@@ -757,42 +757,60 @@ class Node : public Point,  public IndexedObject, public Flags
         typename DofsContainerType::iterator it_begin = mDofs.begin();

```

Comment:
I imagine that we don't need the non const version of this one. This should be used only for fast access but not for filling dof